livingdoc-engine/build.gradle

```diff
@@ -24,3 +30,13 @@ artifacts {
     archives(javadocJar)
     archives(sourcesJar)
 }
+
+buildscript {
```

Comment:
I would like to see the `buildscript` section at the beginning of the file since it is changing the life cycle of this module's build.

Additionally you need to apply the JUnit 5 plugin in order to execute tests during the build. Otherwise Gradle's default test plugin is used, which does not yet recognize JUnit 5 tests.

```
buildscript {
    repositories {
        mavenCentral()
    }
    dependencies {
        classpath 'org.junit.platform:junit-platform-gradle-plugin:1.0.0-M4'
    }
}

apply plugin: 'java'
apply plugin: 'org.junit.platform.gradle.plugin'
```